verify_version.py

```diff
@@ -0,0 +1,15 @@
+import os
+import sys
+
+VERSION = "v0.12.6"  # Ensure this matches the version in pyproject.toml
```

Comment:
@cdoyle-temple I don't think we want this hard coded, then we would have to update it every time.  I think something like this would work better.  add this to the build-system requires - setuptools_scm 
and then set up a task similar to this:
- run:
    name: Verify git tag vs. version.
    command: |
      pip install --user setuptools_scm
      if [ "$(python3 -c 'import setuptools_scm; print(setuptools_scm.get_version())')" != "$(git describe --tags --exact-match 2>/dev/null)" ]; then
        echo "Error: Git tag does not match package version" >&2
        exit 1
      fi